config/katello-answers.yaml

```diff
@@ -41,8 +41,6 @@ foreman::plugin::remote_execution: false
 
 foreman_proxy::plugin::remote_execution::ssh: false
 
-katello::plugin::gutterball: true
```

Comment:
Existing installs won't get this update, so we'll need a migration to remove this from existing answer files too, see `config/migrations/` dir for some examples